src/monodroid/jni/debug-app-helper.cc

```diff
@@ -91,7 +91,7 @@ Java_mono_android_DebugRuntime_init (JNIEnv *env, [[maybe_unused]] jclass klass,
 	void *monosgen = dlopen (monosgen_path, RTLD_LAZY | RTLD_GLOBAL);
 	if (monosgen == nullptr) {
 		log_fatal (LOG_DEFAULT, "Failed to dlopen Mono runtime from %s: %s", monosgen_path, dlerror ());
-		exit (FATAL_EXIT_CANNOT_FIND_LIBMONOSGEN);
```

Comment:
Are all of these `FATAL_EXIT_*` constants somewhere, where we could delete those, too?

Reply:
the `FATAL_EXIT_*` constants are in xamarin/java.interop, some of which are used there.